task/task_smoke_test.go

```diff
@@ -43,6 +91,16 @@ func TestTaskSmoke(t *testing.T) {
 		testName = "smoke test"
 	}
 
+	script := `
+          #!/bin/sh -e
+          test -v TEST_GPU && nvidia-smi
+          mkdir --parents cache output
+          touch cache/file
+          echo "$ENVIRONMENT_VARIABLE_DATA" | tee --append output/file
+          sleep 60
+          cat output/file
+        `[1:]
```

Comment:
Not worth addressing before merging, but this works due to the shell interpreting files without shebang: it's ineffective due to the indentation.